app/routes/circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -130,9 +142,15 @@ export function CircularEditForm({
   const [body, setBody] = useState(defaultBody)
   const [subject, setSubject] = useState(defaultSubject)
   const [format, setFormat] = useState(defaultFormat)
+  const [date, setDate] = useState(createdOnDate)
+  const [time, setTime] = useState(createdOnTime ?? '12:00')
+  const dateValid = circularId ? dateIsValid(date, time) : true
+
+  const [updatedSubmitter, setUpdatedSubmitter] = useState(submitter)
```

Comment:
Rename `submiter` to `defaultSubmitter` and `updatedSubmitter` to `submitter` for consistency with other fields.